cmd/aro/operator.go

```diff
@@ -8,13 +8,15 @@ import (
 	"flag"
 	"fmt"
 
-	machinev1 "github.com/openshift/api/machine/v1"
 	"github.com/sirupsen/logrus"
+
 	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/kubernetes"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 
+	machinev1 "github.com/openshift/api/machine/v1"
```

Comment:
By default it has broken these out into separate import groups since they are under a different org. We can configure the tool not to, but before I spent time on that I figured we might as well see if folks are ok with this approach. Everything is fully automated end-to-end so it's low-touch from here on out.

Reply:
If this behaviour is OpenShift's standard, then I think that's a good reason to keep the defaults.